net/instaweb/rewriter/css_inline_filter.cc

```diff
@@ -107,15 +107,24 @@ void CssInlineFilter::InitStats(Statistics* statistics) {
 }
 
 void CssInlineFilter::StartDocumentImpl() {
+    in_body_ = false;
```

Comment:
also init this in the constructor (I know this is redundant but it just seems cleaner to make sure it doesn't ever have garbage in it).  I think in practice this won't matter.